README.md

```diff
@@ -8,6 +8,10 @@ Install all the dependencies:
 
     $ npm install
 
+Install native packages
+
+    $ npm run install-native-packages
```

Comment:
Oh - I see how it works.  Does the meteor-client bundle -s api pick up the links from the  automatically?